index.bs

```diff
@@ -1662,13 +1662,22 @@ session.SystemProxyConfiguration = (
 
 </pre>
 
-#### The session.UserPromptHandler Type #### {#type-session-UserPromptHandler}
+#### The session.UserPromptHandlerType Type #### {#type-session-UserPromptHandlerType}
 
 [=Remote end definition=] and [=local end definition=]
 
 <pre class="cddl remote-cddl local-cddl">
 session.UserPromptHandlerType = "accept" / "dismiss" / "ignore";
+</pre>
+
+The <code>session.UserPromptHandlerType</code> type represents the behavior
+of the user prompt handler.
+
+#### The session.UserPromptHandler Type #### {#type-session-UserPromptHandler}
```

Comment:
nit: Given that we sort the types, commands and events alphabetically I would switch the order so that this type comes before `session.UserPromptHandlerType`.